src/TestUtils/CloudFunctionDeploymentTrait.php

```diff
@@ -153,7 +158,7 @@ public function getBaseUri()
      * @param callable $process callback function to run on the logs.
      * @param int $retries the number of times to retry entry lookup
      */
-    private function processFunctionLogs(string $startTime, callable $process, int $retries = 3)
+    private function processFunctionLogs(string $startTime, callable $process, int $retries = null)
```

Comment:
I think we're still using `$retries` later in this function (line 183)?

Reply:
yes but it's being passed in as a parameter to another function, which accepts `null` for that parameter.

When `$retries` is null, the property `$eventuallyConsistentRetryCount` is used instead.